src/components/Navbar.js

```diff
@@ -84,7 +84,7 @@ function DrawerAppBar(props) {
                 component={Link}
                 to={`/${item}`.toLowerCase()}
                 key={item}
-                sx={{ color: '#0F2944', fontFamily: 'Proxima Nova', mr: '13px' }}
+                sx={{ bgcolor: item === currPage ? 'rgba(25, 118, 210, 0.04)' : '', color: '#0F2944', fontFamily: 'Proxima Nova', mr: '13px' }}
```

Comment:
Consider making the `bgcolor` a bit darker so it's even more clear which page the viewer is on (it's a bit subtle right now)